tmt/steps/prepare/feature.py

```diff
@@ -32,8 +32,8 @@ def __init__(
 
     def _find_playbook(self, filename: str) -> Optional[Path]:
         filepath = FEATURE_PLAYEBOOK_DIRECTORY / filename
-        if filepath.exists():
-            return filepath
+        if filepath.is_file():
+            return Path(str(filepath))
```

Comment:
I've seen the `str()` call before, is it needed? Can we create `Path()` from, hmm, what exactly is `filepath`? I'd like to avoid having `str()` everywhere for the conversion, if it's needed and `filepath` type is one that would be commonly converted to `Path`, we could add this conversion to `Path` class itself.

Reply:
This is actually just to apease the mypy. `is_file` should technically be `TypeGuard[Path]`. One issue of course is pathlib.Path vs tmt.utils.Path

> Can we create Path() from, hmm, what exactly is filepath

Yes, but there was also a mypy complaint about pathlib.Path being used for tmt.utils.Path. maybe the constructor for the latter needs to be specialized a bit more